ui/core/src/model/units/bytes.ts

```diff
@@ -32,17 +40,14 @@ export const BYTES_GROUP_CONFIG: UnitGroupConfig = {
   abbreviate: true,
 };
 export const BYTES_UNIT_CONFIG: Readonly<Record<BytesUnitKind, UnitConfig>> = {
-  // These units are powers of 1000.
-  // In other words, 1 KB = 1000 bytes.
```

Comment:
Moved this to the top of the file -- make people generally aware of our conventions at the beginning of this file